Readme.md

```diff
@@ -4,20 +4,98 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 [![](https://img.shields.io/badge/💬_Leave_Feedback-feecdd?style=flat-square)](#does-this-example-address-your-development-requirementsobjectives)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* **[MainWindow.xaml](./CS/TBExample/MainWindow.xaml) (VB: [MainWindow.xaml](./VB/TBExample/MainWindow.xaml))**
+# WPF TileBar – Bind Items to a ViewModel Collection (MVVM)
+
+The [`TileBar`](https://documentation.devexpress.com/WPF/115595/Controls-and-Libraries/Navigation-Controls/Tile-Bar) control allows you to create a Windows 10-inspired navigation UI. 
+
+![Bind Items to a ViewModel Collection](./Images/tilebar-view.jpg)
+
+Use the `TileBar` control with MVVM when you need to:
+
+- Define a dynamic navigation UI based on data rather than hard-coded items.
+- Bind a flat or hierarchical data structure to a `TileBar`.
```

Comment:
I want to make sure the word phrase order is correct. In this instance, "flat or hierarchical data structure" is the target and `TileBar` is the source. Is this correct? Or do you mean:

`Bind a `TileBar` to a flat or hierarchical data structure.`

Reply:
Thank you